tests/store/test_rest_store.py

```diff
@@ -62,6 +68,64 @@ def test_response_with_unknown_fields(self, request):
         assert len(experiments) == 1
         assert experiments[0].name == 'My experiment'
 
+    def _verify_requests(self, http_request, host_creds, endpoint, method, json_body):
+        http_request.assert_called_with(host_creds=host_creds,
+                                        endpoint="/api/2.0/preview/mlflow/%s" % endpoint,
+                                        method=method,
+                                        json=json.loads(json_body))
+
+    @mock.patch('requests.request')
+    def test_requestor(self, request):
+        response = mock.MagicMock
+        response.status_code = 200
+        response.text = '{}'
+        request.return_value = response
+
+        creds = MlflowHostCreds('https://hello')
+        store = RestStore(lambda: creds)
+
+        with mock.patch('mlflow.store.rest_store.http_request') as mock_http:
+            store.log_param("some_uuid", Param("k1", "v1"))
+            body = message_to_json(LogParam(run_uuid="some_uuid", key="k1", value="v1"))
```

Comment:
It's a little inconsistent that we rely on the `message_to_json` helper (which the `RestStore` implementation also uses) here, but actually verify the desired string values of the endpoint / HTTP method. However I think it's fair to assume the correctness of `message_to_json` so I'm fine with this (the alternative would be writing out the expected JSON string, e.g. `body = json.loads({"run_uuid": "some_uuid"...})`).

Reply:
we could do that. however, these test are ensuring that `RestStore` is hydrating the API  requests and responses. IMHO, it is okay to rely on `message_to_json` if we can test that method independently.